### ISSUE TITLE
[Request] Adding Gear to the Max Items List

### ISSUE THREAD

**Issue by ghost** [VisualStudio-zz/MLSSE#3]
Personally, I'm using this to see what's left in the game for me that I haven't found yet, since I've been datamining since Day 1. However, since the Info Shop guy has been revamped to show records & give crazy items, I'm trying to see what I missed out on by not 100%'ing the game.

**Comment by GoobyCorp** [comment]
Gear is odd, it's in random spots of the save and rather hard to locate what unlocks what.

**Comment by ghost** [comment]
Makes sense. Because that's the same with other games like Dream Team & Paper Jam, no? Correct me if I'm wrong on that, but I figured that would be the case with all of the 3DS M & L's.
